prbench-models/src/prbench_models/geom2d/envs/motion2d/parameterized_skills.py

```diff
@@ -42,7 +42,8 @@ def __init__(
     def sample_parameters(
         self, x: ObjectCentricState, rng: np.random.Generator
     ) -> tuple[float, float, float]:
-        # Sample relative position within the target region
+        del x  # Unused
```

Comment:
this shouldn't be necessary because `sample_parameters` is overriding a method from the parent class